### ISSUE TITLE
[Snare and root resist]

### ISSUE THREAD

**Issue by Piranhaz** [ccshiro/cc-buglist#2457]
The effects from the meta gem "enigmatic skyfire diamond" and the shoes enchantment "surefooted"
does not work at all. Dueled 10-20 times as a warrior against a frost mage. Neither frostnova nor the elemental nova had been resisted.

**Comment by ccshiro** [comment]
I've noticed this meta gem working just a few weeks ago. You sure you're not confusing how it should work? You're probably not going to see frost nova as resisted in recount, for example, the root won't apply but the damage will (which means you didn't resist the spell).

**Comment by Piranhaz** [comment]
Yeah it's about the root effect only but i try to find someone to test it to get some % data of the resists.
